conf/aggregations.go

```diff
@@ -9,16 +9,10 @@ import (
 	"github.com/alyu/configparser"
 )
 
-var DefaultAggregation = Aggregation{
-	Name:              "default",
-	Pattern:           nil,
-	XFilesFactor:      0.5,
-	AggregationMethod: []Method{Avg},
-}
-
 // Aggregations holds the aggregation definitions
 type Aggregations struct {
-	Data []Aggregation
+	Data               []Aggregation
+	DefaultAggregation Aggregation
```

Comment:
DefaultAggregation is only modified by unitTests.  Keeping it as part of the Aggregations{} instance makes it easier for unit tests to modify the defaults for testing purposes.

eg, calling mdata.SetSingleAgg(met ...conf.Method) wont break other unit tests that may be exacting the DefaultAggregation to be set to the defaults.